src/plugins/shared_ux/public/components/no_data_views_page/assets/no_results_illustration.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import './no_results_illustration.scss';
+import React from 'react';
+
+export const NoResultsIllustration = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="226"
+    height="166"
+    fill="none"
+    viewBox="0 0 226 166"
+  >
+    <path
+      className="dscNoResultsIllustration__decor"
```

Comment:
these need to be changed so they don't reference Discover, but didn't want to do this yet in case we're not going to go with this particular `svg` in the end.